Enhance signature pdf export

mllocs opened the pull request:
#### :tophat: What? Why?

Improve the format of the PDF export of the initiatives votes signatures to make it more useful. 

#### :pushpin: Related Issues
- Closes https://github.com/decidim/decidim/issues/13685

#### Testing
- Export the PDF Signatures from the admin edit page of an initiative https://decidim-lot2.populate.tools/admin/initiatives/i-1/edit

### :camera: Screenshots

![CleanShot 2024-12-17 at 18 08 33@2x](https://github.com/user-attachments/assets/f84cf3fe-dbb8-4b97-b707-303b54f9ef11)

:hearts: Thank you!

mllocs commented:
@NilHomedes sending it to you, @furilo already sent me feedback in another channel.

NilHomedes reviewed (changes requested):
Hi @mllocs and happy new year! 

Everything is working as expected, just a thing:

- [ ] The PDF export file is showing empty columns. So, as defined in the AC, if we are not collecting personal data, then the columns: Name and surname, Document number, Date of birth, Postal code. Should not be displayed in the export file. 
![imatge](https://github.com/user-attachments/assets/c4b01330-8366-4274-8a31-e0a98b23ea42)

mllocs commented:
> Hi @mllocs and happy new year!
> 
> Everything is working as expected, just a thing:
> 
> * [ ]  The PDF export file is showing empty columns. So, as defined in the AC, if we are not collecting personal data, then the columns: Name and surname, Document number, Date of birth, Postal code. Should not be displayed in the export file.
>   ![imatge](https://private-user-images.githubusercontent.com/71442847/401082997-c4b01330-8366-4274-8a31-e0a98b23ea42.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYzMzI2NjgsIm5iZiI6MTczNjMzMjM2OCwicGF0aCI6Ii83MTQ0Mjg0Ny80MDEwODI5OTctYzRiMDEzMzAtODM2Ni00Mjc0LThhMzEtZTBhOThiMjNlYTQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA4VDEwMzI0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZjNDJhOTU4ZTM2ZjljMWQyYmI1YTgxYWRhZGY1NDYyMGNmMDIxOGZlYmQwNzQxOGVhMjlmNzJkZTczY2Y1ZGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.YxkdJBqhQmkMrtR_mpPGwuC3WkH5sURF6RVa7AvKP_A)

Hey @NilHomedes! Happy new year!

Yep, noticed that. The thing is that depending on the Initiative Type, the admin decides if these fields are collected or not (See "Collect participant personal data on signature" in https://decidim-lot2.populate.tools/admin/initiatives_types/1/edit). I'm showing the columns or not depending on that setting, the thing is that those fields are not mandatory and can still be empty...

NilHomedes reviewed (changes requested):
Hey @mllocs, I disabled the setting "Collect personal data", and now the PDF file is in vertical format.  See: [votes_1-2.pdf](https://github.com/user-attachments/files/18376948/votes_1-2.pdf)


- [ ] The signature PDF file should always be horizontal.
- [ ] Could we get rid of the modal that appears when clicking on "Export signatures"? It seems that you are deleting something, but that's not the case for the action of exporting signatures.
![Screenshot 2025-01-10 at 13-37-37 About this initiative - Soterrament de les vies de Montcada Estació - Admin - Test Site for Lot2](https://github.com/user-attachments/assets/d5745b2f-bb1c-4f6d-b5aa-34bfb110773f)



- [ ]

mllocs commented:
> Hey @mllocs, I disabled the setting "Collect personal data", and now the PDF file is in vertical format. See: [votes_1-2.pdf](https://github.com/user-attachments/files/18376948/votes_1-2.pdf)
> 
> * [ ]  The signature PDF file should always be horizontal.
> * [ ]  Could we get rid of the modal that appears when clicking on "Export signatures"? It seems that you are deleting something, but that's not the case for the action of exporting signatures.
>   ![Screenshot 2025-01-10 at 13-37-37 About this initiative - Soterrament de les vies de Montcada Estació - Admin - Test Site for Lot2](https://private-user-images.githubusercontent.com/71442847/401989434-d5745b2f-bb1c-4f6d-b5aa-34bfb110773f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY3NzUwMTYsIm5iZiI6MTczNjc3NDcxNiwicGF0aCI6Ii83MTQ0Mjg0Ny80MDE5ODk0MzQtZDU3NDViMmYtYmIxYy00ZjZkLWI1YWEtMzRiZmIxMTA3NzNmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTEzVDEzMjUxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWViNDBkMGU3MzUxMWM5NjVhNmQ5ZGQwYWVhYjU3NmE3MjUyZjJlODQ5ZWZlMmNmOTgwZjdkYmQ5NDZiODc2NjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qaYweVccEEpH9DvhYzNn1hzMcOyUhpy6IvrTV1ZVkGc)
> * [ ]

Fixed both items! Updated staging :)

alecslupu reviewed (approved):
Codewise looks ok, I am waiting @NilHomedes's input to move forward.

NilHomedes reviewed (approved):
Thanks @mllocs now looks good to me! :100:
